src/wazuh_modules/wm_github.c

```diff
@@ -98,16 +111,31 @@ void * wm_github_main(wm_github* github_config) {
         mtinfo(WM_GITHUB_LOGTAG, "Module GitHub disabled.");
     }
 
+#ifdef WIN32
+    return 0;
+#else
     return NULL;
+#endif
 }
-
+#ifdef WIN32
+STATIC DWORD WINAPI wm_github_destroy(void* github_config) {
+    wm_github *ptr = (wm_github *)github_config;
+    mtinfo(WM_GITHUB_LOGTAG, "Module GitHub finished.");
+    wm_github_auth_destroy(ptr->auth);
+    wm_github_fail_destroy(ptr->fails);
+    os_free(ptr->event_type);
+    os_free(github_config);
+    return 0;
```

Comment:
this could be summarized with a #ifdef WIN32 on the return to avoid the entire duplication of the function.